sled-agent/src/instance_manager.rs

```diff
@@ -325,6 +328,20 @@ impl InstanceManager {
             .map_err(|_| Error::FailedSendInstanceManagerClosed)?;
         rx.await?
     }
+
+    /// Marks instances failed unless they're using "disks".
```

Comment:
The scare quotes around "disks" are very amusing to me but I'm not _entirely_ sure what it signifies in this context?

Reply:
Ahhhh lazy rustdocs. This is the name of the argument, I'll use back-ticks to make this more clear.